intel_extension_for_transformers/llm/runtime/graph/scripts/convert_gptq_mistral.py

```diff
@@ -83,6 +83,10 @@ def main(args_in: Optional[List[str]] = None) -> None:
 
     f.write(struct.pack("f", config["rms_norm_eps"]))
     f.write(struct.pack("f", config["rope_theta"] if "rope_theta" in config else 10000))
+    rope_scale = 1
+    if "rope_scaling" in config and config["rope_scaling"] is not None:
+        rope_scale = config["rope_scaling"]["factor"] if "factor" in config["rope_scaling"] else 1
```

Comment:
implementation quite confusing
```suggestion
rope_scale = config.get("rope_scaling", {}).get("factor", 1)
```

Reply:
```
llama/Llama-2-7b-chat-hf/config.json                                                                                                                              
19:  "rope_scaling": null,

Magicoder-S-DS-6.7B/config.json                                                                                                                                   
21:  "rope_scaling": { 
  "factor": 4.0,
```
other models do not have this parameter

Reply:
`config.get("rope_scaling", {})` will return None, None can not `.get("factor", 1)`

Reply:
> `config.get("rope_scaling", {})` will return None, None can not `.get("factor", 1)`

I just tried, anything missing?
```
(base) hengyume@DUT7050PVC:~/neural-speed/build> python
Python 3.9.18 (main, Sep 11 2023, 13:41:44) 
[GCC 11.2.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> {}.get("a", 1)
1
>>> {"a": 1}.get("b", {}).get("c", 1)
1
```

Reply:
> "rope_scaling": null,

`config.get("rope_scaling", {})` will return None, not {}

Reply:
get it, how about 
```suggestion
    rope_scale = 1
    if config.get("rope_scaling") is not None:
        rope_scale = config["rope_scaling"].get("factor", 1)
```

Reply:
great! 
need change to one line?
`rope_scale = config["rope_scaling"].get("factor", 1) if config.get("rope_scaling") is not None else 1`

Reply:
one-line seems complicated

Reply:
updated